system-addon/lib/ActivityStream.jsm

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+Components.utils.import("resource:///modules/NewTabURL.jsm");
+
+class ActivityStream {
+  constructor(options) {
+    this.initialized = false;
+    this.options = options;
+  }
```

Comment:
There wants to be some JSDoc somewhere explaining what options the constructor expects.